src/tool/cppwinrt/cppwinrt/type_writers.h

```diff
@@ -71,12 +71,20 @@ namespace xlang
     {
         using writer_base<writer>::write;
 
+        struct depends_compare
+        {
+            bool operator()(TypeDef const& left, TypeDef const& right) const
```

Comment:
noexcept

Reply:
The trouble is that the `TypeDef`'s `TypeName` and `TypeNamespace` members are not `noexcept` so any error retrieving those values would not propagate to a console error message but would instead crash the compiler.